docs/CONTRIBUTING.md

```diff
@@ -34,6 +34,22 @@ To set up your development environment for firewatch, please follow these steps:
 5. Run pre-commit hooks to ensure code quality: `make pre-commit`
 6. Execute tests to ensure everything is working correctly: `tox` or `make test`
 
+## Testing Changes
+
+1. Find a failed prow job you would like to test against.
+    - **Example**: [periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws #1696039978221441024](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws/1696039978221441024)
+2. Fill out `firewatch/development/env.list`
+   - `JIRA_TOKEN`: The token needed to log in to the Jira service account that firewatch will use.
+   - `JIRA_SERVER`: URL to Jira server you would like to test against (should be `https://issues.stage.redhat.com`)
+   - `BUILD_ID`: Build ID from failed job you are testing with (`1696039978221441024` if using job from the example in step 1)
+   - `JOB_NAME_SAFE`: Safe job name from the prow job you are testing with (`openshift-pipelines-interop-aws` if using the job from the example in step 1)
+   - `JOB_NAME`: Full prow job name that you are testing with (`periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws` if using the job from the example in step 1)
+   - `FIREWATCH_CONFIG`: Config you would like to test with. Use the [configuration guide](configuration_guide.md) for help.
+   - `FIREWATCH_DEFAULT_JIRA_PROJECT`: Default project you'd like to report failures to if a rule doesn't match a failure.
+3. From the root of this repository, run `make docker-build-test` to execute `firewatch report` using the values provided above.
```

Comment:
Maybe add how to run without container as well? (poetry) useful for develop and debugging

Reply:
I can try, this is the first time I have ever used poetry, so it's probably going to take some time to understand